packages/libs/react-ui/src/components/Dialog/Dialog.tsx

```diff
@@ -10,18 +10,37 @@ import { useOverlayTriggerState } from 'react-stately';
 import type { IModalProps } from '../Modal/Modal';
 import { Modal } from '../Modal/Modal';
 import { DialogContext } from './Dialog.context';
-import { closeButtonClass, overlayClass } from './Dialog.css';
+import {
+  closeButtonClass,
+  lgClass,
+  mdClass,
+  overlayClass,
+  smClass,
+} from './Dialog.css';
 
+const classMap = {
+  sm: smClass,
+  md: mdClass,
+  lg: lgClass,
+};
 interface IBaseDialogProps
   extends Omit<IModalProps, 'children'>,
     AriaDialogProps {
   children?: ((state: OverlayTriggerState) => ReactNode) | ReactNode;
   className?: string;
+  size?: 'sm' | 'md' | 'lg';
```

Comment:
I suggest to keep it a bit more dynamic, for example call the classMap sizeMap and use the keyof typeof sizeMap here.

Reply:
i like it but since i didnt export it i thought this would make it faster to spot the values, but im okay with doing this